pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake
+, libjpeg ? null
+, zlib ? null
+, libpng ? null
+, eigen ? null
+, libtiff ? null
```

Comment:
Do note that `callPackage` does pass flags that have default value (i.e. the default is currently to use all the optional dependencies, not to use the minimal set). General NixPkgs decision making on optional dependencies is based on «how often extra features are useful» vs. «total closure size change», and include-everything is quite a typical outcome (no change required here, just giving some context)